src/pages/commits/ActiveCommits/utils.test.ts

```diff
@@ -12,8 +12,8 @@ import {
   getStatusesWithZeroCount,
   constructBuildVariantDict,
   roundMax,
```

Comment:
Should the tests for `injectGlobalHighlightStyle` and `removeGlobalHighlightStyle` have been removed from this file? I thought it might make sense to have tests for all four functions (unless that seems too redundant)